grpc/pb-marshalling.go

```diff
@@ -234,3 +237,112 @@ func argsToPerformValidationRequest(domain string, challenge core.Challenge, aut
 	}, nil
 
 }
+
+func registrationToPB(reg core.Registration) (*rapb.Registration, error) {
```

Comment:
Are you open to writing some unit tests for the `xToPB` and `pbToX` functions you're adding in this PR? Something similar to `TestChallenge` from `grpc/pb-marshalling_test.go` for the registration and authz converters